### DISCUSSION TITLE
Question About Timeout

### DISCUSSION THREAD

**Discussion by wang975679801** [PaddlePaddle/PaddleSpatial#100] [BaiduKDDCup2022] [answered]
Our method gets the running timeout error. Please give us more time for online test!

**Comment by ramber1836** [comment]
Currently, it is 3 hours. We will prolong it and update the online test.
We will let you know when it is ready.

**Comment by zhoujingbo** [comment, marked as answer]
> 

Hi, Please note that we cannot guarantee very long running time be acceptable in this competition. In real-life applications, the test time also cannot be too long.

**Reply by wang975679801** [reply]
If load the test data in a larger batch (Batchsize * Sequence Length * Feature Dimension) , the time cost would be acceptable. However, in recent evaluation protocol, the TestData function load the data with batchsize=1. Therefore, I think it's rational to set longer test time.

**Comment by ramber1836** [comment]
@wang975679801 
The timeout setting has been currently updated (10h at most), please try it again.
